frontend/src/metabase/query_builder/components/filters/pickers/DefaultPicker.jsx

```diff
@@ -178,7 +176,7 @@ export default function DefaultPicker({
 
   return (
     <DefaultPickerContainer
-      limitHeight={!isSidebar}
+      limitHeight
```

Comment:
nit: This is the only place that `DefaultPickerContainer` is used. So, it should be safe to modify their definition to not rely on this `limitHeight` prop anymore

https://github.com/metabase/metabase/blob/6c43ace1d1be0a133fcd487f29f33105af40427f/frontend/src/metabase/query_builder/components/filters/pickers/DefaultPicker.styled.tsx#L26-L34